prow/jobs_config/knative/infra.yaml

```diff
@@ -0,0 +1,11 @@
+org: knative
+repo: infra
+branches: [main]
+image: gcr.io/knative-tests/test-infra/prow-tests:stable
+imagePullPolicy: Always
+
+jobs:
+  - name: unit-tests
+    types: [presubmit]
+    command: [runner.sh, ./test/presubmit-tests.sh, --unit-tests]
```

Comment:
This is running a script that is dependent on one script in the knative/hack repo, so this means you'll need to vendor knative/hack with go.mod, are you sure it's desired? :-)